jupyterhub/notebook-images/overlays/additional/spark-scipy-notebook-imagestream.yaml

```diff
@@ -3,6 +3,7 @@ kind: ImageStream
 metadata:
   labels:
     opendatahub.io/notebook-image: "true"
+    opendatahub.io/notebook-image-name: "Minimal Python with Apache Spark and SciPy"
```

Comment:
This should be an `annotation`. See [generic-data-science notebook](https://github.com/opendatahub-io/odh-manifests/blob/master/jupyterhub/notebook-images/overlays/additional/generic-data-science-notebook-imagestream.yaml#L8)